packages/matchbox/src/components/LabelValue/LabelValue.js

```diff
@@ -49,6 +56,7 @@ LabelValue.propTypes = {
   children: PropTypes.node,
   className: PropTypes.string,
   orientation: PropTypes.oneOf(['horizontal', 'vertical']),
+  appearance: PropTypes.oneOf(['inverted']),
```

Comment:
I think we need the normal version too, not sure what to call it. `default`? or `normal`?

Reply:
Should also set a default prop value